ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/inventory/group_vars/DC1.yml

```diff
@@ -1,6 +1,14 @@
 ---
 dc_name: DC1
 
+bgp_peer_groups:
+  IPv4_UNDERLAY_PEERS:
+    password: "AQQvKeimxJu+uGQ/yYvv9w=="
+  EVPN_OVERLAY_PEERS:
+      password: "q+VNViP5i4rVjW1cxFv2wA=="
```

Comment:
Nitpicking, but indentation looks off here.

Reply:
Thanks ! Good catch. It is now fixed.